_posts/2020-12-14-tenants-rights-guide.md

```diff
@@ -0,0 +1,90 @@
+
+{: style="text-align: center" }
+## Common Legal Defenses Against Eviction
+
+**Bad conditions**
+-   Do you have pests? Does the plumbing leak? Are any locks broken? Does cold air seep in through the windows? If your apartment has problems or, and your landlord knows or should know about them, you are owed for the loss in value of the home. You have the right to a safe, sanitary home. You can call your local Inspectional Services Department for more information about the sanitary code or to request an inspection.
+
+**Retaliation**
+-   Is your landlord trying to get rid of you because you complained, organized a tenant’s union, or other-wise exercised your rights? Your landlord cannot retaliate against tenants who exercise their rights, and the burden of proof is on the landlord.
+
+**Discrimination**
+-   Is your landlord singling you out because of race, religion, gender, sexual orientation, disability, national origin, age, or another protected identity?
+
+**Interference with the quiet enjoyment of your home**
+-   Does the landlord barge into your apartment in non-emergency situations without warning you? Did they change the locks on you, or put your stuff out onto the street? Depending on the facts of the case, this may count as an illegal eviction!
+
+{: style="text-align: center" }
+## Federal/CDC Eviction Ban
+
+The Center for Disease Control has ordered the suspension of physical evictions for non-payment of rent until December 31, 2020. If you meet all 5 points of criteria below, you can submit the CDC declaration form to your landlord to protect yourself from eviction and to use as a legal tool in court:
+
+-   Used “best efforts” to look for government assistance for rent or housing. (This is vague, but if you can prove that you did everything you can to pay the back rent or find alternate housing arrangements, you should be able to meet this requirement.)
+-   Meet one of these income requirements:
+    *  You will earn less than $99,000 this year OR if filing with a partner or household, you will earn less than $198,000.
+    *  You were not required to pay taxes in 2019.
+    *  You qualified for an income stimulus check earlier this year.
+-   Are unable to make your full rent payment because of a loss of income or large medical expense.
+-   Are doing your best to make timely partial rental payments. If you haven’t already been paying your landlord, offer a partial rent payment that is affordable to you when you give your landlord the CDC notice.
+-   Would either become homeless or enter a shared living space (like living with a family member or staying at a shelter) if evicted.
+
+Confused about the CDC moratorium process? Follow these five steps to apply for the moratorium:
+
+1. Ask for help with rent!
+   *  Boston: [metrohousingboston.org](metrohousingboston.org)
```

Comment:
These links are not valid, they should should be:
- https://metrohousingboston.org/
- https://masshousinginfo.org

If you don't include `https://`, the visitor browsers will think it's a local link and it won't work